src/xdocs/google_style.xml

```diff
@@ -664,6 +664,12 @@
                     <img src="images/ok_blue.png" alt="" />
                   </span>
                   <a href="checks/blocks/rightcurly.html#RightCurly">RightCurly</a>
+                  <br/>
+                  <br/>
+                  The "Exception" part of the rule is not satisfied, it is addressed in the issue
+                  <a href="https://github.com/checkstyle/checkstyle/issues/15324">
+                    #15324
```

Comment:
in this comment https://github.com/checkstyle/checkstyle/pull/14901/files#r1660867034
we reference different issue.
please clarify.

Reply:
Both issues are similar and refers to the same problem, my bad I shouldn't have created new issue for this.

Reply:
changed the issue reference:

https://github.com/checkstyle/checkstyle/blob/57dfc92e25024546e39d27870f70a6edca707f59/src/xdocs/google_style.xml#L669-L671